support-workers/src/test/scala/com/gu/support/workers/EndToEndSpec.scala

```diff
@@ -22,8 +22,9 @@ class EndToEndSpec extends LambdaSpec {
   they should "work with other currencies" in runSignupWithCurrency(EUR)
 
   def runSignupWithCurrency(currency: Currency) {
-    SafeLogger.info(createPayPalPaymentMethodContributionJson(currency))
-    val output = wrapFixture(createPayPalPaymentMethodContributionJson(currency))
+    val json = createStripePaymentMethodContributionJson(currency = currency)
```

Comment:
I changed to stripe as once it was actually trying the subscribe, it turned out the paypal test BAID is expired.  Since stripe has a permanent test token, I decided to use that instead.